### PR TITLE
Prevent processing logs to client-side-events project

### PR DESCRIPTION
**Pull request by stulees**
## Description
Prevent the flow of processing sending logs to client-side-events project.

This is the simple and safe set of changes to stop logging to client-side-events. It allows for the easiest and safest reversion in case reverting becomes necessary.  

A follow up PR is planned to address removing the full scope of codebase associated with logging to client-side-events and will be initiated/merged when it's confirmed we can do so - `Display_Events.events` and `Display_Events.events_test` tables are deleted. 

## Motivation and Context
Issue #158 

## How Has This Been Tested?
Tested using Charles Proxy to map to local version of common-template and config-prod and ran this schedule on ChrOS player. Validated no logs to `Display_Events.events` were inserted. 

 - [X] Common template has been tested on Player
 - [ ] Common template has been tested in Apps Template Editor

## Release Plan:
- As the Submitter, upon requesting review of this pull request, I confirm that the [Release Checklist](https://help.risevision.com/hc/en-us/articles/360031119991) has been completed.
- As the Reviewer, upon approving the changes in this PR, I confirm I have reviewed and I agree that the [Release Checklist](https://help.risevision.com/hc/en-us/articles/360031119991) has been completed

#### Release Checklist Items Skipped?
no

### CODE REVIEW

File: src/rise-logger.js [hunk 1]
```diff
@@ -258,7 +258,7 @@ RisePlayerConfiguration.Logger = (() => {
       `HTML Template: ${templateData.name}` :
       componentData.name;
 
-    RisePlayerConfiguration.Viewer.sendEndpointLog({
+    return RisePlayerConfiguration.Viewer.sendEndpointLog({
       severity: level,
       eventErrorCode: ( eventDetails && eventDetails.errorCode ) || ( level === "error" ? "X" : null ),
       eventApp,
```
Comment (santiagonoguez) [inline src/rise-logger.js:264]: @stulees as coveralls is failing, maybe commenting the lines above will help, but then the functions called by these would also need to be commented out mabye :'(

**Review by santiagonoguez — COMMENTED** [review]
Looks good to me, just curious what you'll decide for the coveralls error.

File: src/rise-logger.js (hunk 1 above)
Comment (stulees) [reply src/rise-logger.js:264]: @santiagoarriaga Yes it gets messy doing that. I think its cleaner to take the coveralls drop for now and as mentioned it should recover in the future PR that does full removal.

File: src/rise-logger.js [hunk 2]
```diff
@@ -1,5 +1,5 @@
 /* global TEMPLATE_COMMON_CONFIG */
-/* eslint-disable no-console, one-var */
+/* eslint-disable no-console, one-var, no-unreachable */
```
[the remark this answers is not in the document]
Comment (stulees) [reply src/rise-logger.js:2]: @olegrise Good point, will do

[the remark this answers is not in the document]
Comment (stulees) [reply src/rise-logger.js]: @olegrise Done, see last commit, thanks

**Review by santiagonoguez — APPROVED** [review]

**Review by olegrise — APPROVED** [review]

**Pull request merged**